### DISCUSSION TITLE
How to add static files to distributable

### DISCUSSION THREAD

**Discussion by d-bucur** [antfu/vitesse-webext#148] [Q&A]
I am trying to add a CSS file that is loaded dynamically at runtime, but I can't seem to get it to package into the distributable. I've tried putting the file inside /assets hoping it will get packaged as a static file, but it doesn't if there is no import or static reference to it in the code. How are you supposed to just package a static file?

**Comment by Elliot67** [comment]
If you want to pack a static file in the final build, you need to add the file inside `extension/assets`.
